packages/table/src/headers/_headers.scss

```diff
@@ -254,3 +252,30 @@
   }
   // stylelint-enable selector-class-pattern
 }
+
+.bp-table-column-name,
+.bp-table-row-name {
+  transition: color $cell-transition-duration;
+}
+
+.bp-table-header.pt-loading {
+  @include vertical-center();
+
+  .bp-table-column-name,
+  .bp-table-row-name {
+    flex: 1;
+  }
+
+  .bp-table-column-name-text {
+    @include vertical-center();
+    padding: ($pt-grid-size * 1.1) $pt-grid-size ($pt-grid-size * 1.2);
```

Comment:
it does center the children content of `.bp-table-column-name-text`. the default header is 30px high. the problem with a 7px height skeleton is that we need unequal top / bottom padding to make it 30px (i.e. 11px top, 22px bottom)